src/litserve/server.py

```diff
@@ -472,3 +472,68 @@ def setup_auth(self):
         if LIT_SERVER_API_KEY:
             return api_key_auth
         return no_auth
+
+
+def run_all(
+    servers: List[LitServer],
+    port: Union[str, int] = 8000,
+    num_api_servers: Optional[int] = None,
+    log_level: str = "info",
+    generate_client_file: bool = True,
+    api_server_worker_type: Optional[str] = None,
+    **kwargs,
+):
+    """Run multiple LitServers on the same port."""
+    if not servers:
+        raise ValueError("No servers provided to run_all")
+
+    if any(not isinstance(server, LitServer) for server in servers):
+        raise ValueError("All elements in the servers list must be instances of LitServer")
+
+    if generate_client_file:
+        for server in servers:
+            server.generate_client_file()
```

Comment:
let's only call this once

Reply:
Sure @aniketmaurya, I'll check out the docs as well. 
Thanks for the suggestion! 🙌